feat(currency-inr): INR added to supported currencies

StarWar opened the pull request:
Hi, as discussed over https://meta.discourse.org/t/discourse-subscriptions-plugin/140818/18, This PR  adds INR to the currency list.

rimian commented:
Looks good. But you've also changed the coding standards. I'd rather see only the changes made. Not linting.

StarWar commented:
@rimian Sorry for that, that was not intensionally done, I'm currently working on some projects in ruby and learning vuejs on my machine. my editor vscode, again and again, reverted my changes and same was done by Robocop, as the format on save was enabled. Please, run linters on your machine this time, they'll easily revert my changes. Next time, I'll keep in mind to turn off linting or use the config provided by the project only and don't alter the coding standard. I have one issue with current solution though, that is, it displays both currency name and symbol. It should either show currency or symbol in the case of INR, not both. However, with some jquery, I managed to remove that on the subscribe page, e.g. here https://www.rajputcommunity.in/s

StarWar commented:
@rimian I've reverted changes as per existing coding style and removed a hard-coded '$' symbol.  

Also, as I understand this plugin still doesn't support 3D Secure Auth Flow, when will it be added? As of now, no transactions are working successfully with any of my cards. However, this PR still adds an option to INR currency and its symbol. Can you merge it now?
